mergeksortedlists.py

```diff
@@ -0,0 +1,51 @@
+# Time Complexity : O(NlogK)
+# Space Complexity : O(K)
+# Did this code successfully run on Leetcode : No, need to figure out how to implement comparater in python
+# Any problem you faced while coding this : Yes, Comparator implementation in python
```

Comment:
I'll check if I can find some resources that can help you with the comparator implementation.